esp/esdllib/esdl_script.cpp

```diff
@@ -1968,20 +2096,54 @@ static inline void replaceVariable(StringBuffer &s, IXpathContext *xpathContext,
     }
 }
 
+class CEsdlFunctionRegister : public CInterfaceOf<IEsdlFunctionRegister>
```

Comment:
This is currently an internal Interface / class that does exactly what the name implies.  It can easily evolve into something larger in the future and can be renamed.